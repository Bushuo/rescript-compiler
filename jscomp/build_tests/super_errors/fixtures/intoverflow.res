let v1: int = 2_147_483_647
let v2: int = 2_147_483_648
let v3: int = 2_147_483_649
let v4: int = -2_147_483_647
let v5: int = -2_147_483_648
let v6: int = -2_147_483_649
