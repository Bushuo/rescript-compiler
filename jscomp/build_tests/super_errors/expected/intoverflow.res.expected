
  [1;33mWarning number 107[0m
  [36m/.../fixtures/intoverflow.res[0m:[2m2:15-27[0m

  1 [2mâ”‚[0m let v1: int = 2_147_483_647
  [1;33m2[0m [2mâ”‚[0m let v2: int = [1;33m2_147_483_648[0m
  3 [2mâ”‚[0m let v3: int = 2_147_483_649
  4 [2mâ”‚[0m let v4: int = -2_147_483_647

  Integer literal exceeds the range of representable integers of type int


  [1;33mWarning number 107[0m
  [36m/.../fixtures/intoverflow.res[0m:[2m3:15-27[0m

  1 [2mâ”‚[0m let v1: int = 2_147_483_647
  2 [2mâ”‚[0m let v2: int = 2_147_483_648
  [1;33m3[0m [2mâ”‚[0m let v3: int = [1;33m2_147_483_649[0m
  4 [2mâ”‚[0m let v4: int = -2_147_483_647
  5 [2mâ”‚[0m let v5: int = -2_147_483_648

  Integer literal exceeds the range of representable integers of type int


  [1;33mWarning number 107[0m
  [36m/.../fixtures/intoverflow.res[0m:[2m6:16-28[0m

  4 [2mâ”‚[0m let v4: int = -2_147_483_647
  5 [2mâ”‚[0m let v5: int = -2_147_483_648
  [1;33m6[0m [2mâ”‚[0m let v6: int = -[1;33m2_147_483_649[0m
  7 [2mâ”‚[0m 

  Integer literal exceeds the range of representable integers of type int